<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordSafeWidget</class>
 <widget class="QWidget" name="PasswordSafeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>62</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PasswordSafeWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="lockUnlockButton">
     <property name="text">
      <string>U</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="editButton">
     <property name="text">
      <string>E</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="removeButton">
     <property name="text">
      <string>D</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
